Source/ACE.Server/WorldObjects/Player_Trade.cs

```diff
@@ -161,6 +162,52 @@ public void HandleActionAcceptTrade(Session session, ObjectGuid whoAccepted)
 
                 if (session.Player.TradeAccepted && target.TradeAccepted)
                 {
+                    var playerAitems = new List<WorldObject>();
+                    var playerBitems = new List<WorldObject>();
+
+                    foreach (ObjectGuid itemGuid in session.Player.ItemsInTradeWindow)
+                    {
+                        var wo = session.Player.GetInventoryItem(itemGuid);
+
+                        if (wo == null)
+                            wo = session.Player.GetEquippedItem(itemGuid);
+
+                        if (wo != null)
+                            playerAitems.Add(wo);
+                    }
+
+                    foreach (ObjectGuid itemGuid in target.ItemsInTradeWindow)
+                    {
+                        var wo = target.GetInventoryItem(itemGuid);
+
+                        if (wo == null)
+                            wo = target.GetEquippedItem(itemGuid);
+
+                        if (wo != null)
+                            playerBitems.Add(wo);
+                    }
+
+                    var playerACanAddToInventory = session.Player.CanAddToInventory(playerBitems);
```

Comment:
what would happen in this scenario?

both players have every pack slot full, and they each try to trade 1 item with each other

Reply:
As coded, trade would fail. It doesn't subtract inventory/burden from items the trade partner is trading away from their inventory